test/parallel/test-repl-tab-complete.js

```diff
@@ -359,6 +359,139 @@ testMe.complete('var log = console.lo', common.mustCall((error, data) => {
   assert.deepStrictEqual(data, [['console.log'], 'console.lo']);
 }));
 
+// Test tab complete for fs module
+putIn.run(['.clear']);
+
+// does not auto complete empty paths
```

Comment:
Please always use a capital letter as first character in a comment and punctuate them.